lib/generators/talking_stick/views/views_generator.rb

```diff
@@ -0,0 +1,22 @@
+class TalkingStick::ViewsGenerator < Rails::Generators::Base
+  source_root File.expand_path('../templates', __FILE__)
```

Comment:
What if you set the `source_root` to `copy_path`, something like this:

```
class TalkingStick::ViewsGenerator < Rails::Generators::Base
  source_root File.expand_path('../../../../../app/views/talking_stick', __FILE__)

  def generate_participant_views
    directory "participants", "#{paste_path}/participants"
  end

  def generate_room_views
    directory "rooms", "#{paste_path}/rooms"
  end

private

  def paste_path
    'app/views/talking_stick'
  end
end
```

Does that work? I don't think the current `source_root` is valuable.